### PR TITLE
[ENDPOINT] OT226-37 authenticated user data

### PR DESCRIPTION
**Pull request by raoole20**
[OT226-37](https://alkemy-labs.atlassian.net/browse/OT226-37)
### Descrition

I WANT to get my data
TO have knowledge of the information that I store in the system

Criteria of acceptance:
/GET auth/me - They must return the data of the authenticated user, obtaining it from the token sent in the Authorization header

### Evidence 

**success msg**
image

### CODE REVIEW

**Review by Franckfer — APPROVED** [review]
Excelente PR!

File: controllers/users.js [hunk 1]
```diff
@@ -104,4 +106,21 @@ module.exports = {
       next(httpError)
     }
   }),
+  getInformation: catchAsync(async (req, res, next) => {
```
Comment (gabodmc) [inline controllers/users.js]: Este getInformation me hace un poco de ruido. Iria por algo un poco mas informativo tipo getUserByToken o algo asi.

**Review by gabodmc — CHANGES_REQUESTED** [review]
Estan muy bien los cambios. Solo agrego una sugerencia

**Review by gabodmc — APPROVED** [review]

**Review by dangello-g — APPROVED** [review]
Buen PR

**Pull request merged**